app/estimate_square_size.go

```diff
@@ -151,7 +151,7 @@ func rawShareCount(txs []*parsedTx, evd core.EvidenceList) (txShares, evdShares
 		namespace []byte
 	}
 
-	var msgSummaries []msgSummary
+	msgSummaries := make([]msgSummary, 0)
```

Comment:
[blocking nit]
we should probably just use the `//nolint:prealloc` flag here to be explicit.

Reply:
Addressed this comment and amend the same commit. Let me know if any concern ^^